packages/crypto/src/hashing.ts

```diff
@@ -47,18 +47,21 @@ async function hashTransaction(tx: TxBody, encoding: 'bytes', includeSign?: bool
 async function hashTransaction(tx: TxBody, encoding = 'base64', includeSign = true): Promise<Buffer | string> {
     // check amount format
     tx.amount = '' + tx.amount;
+    const amount = tx.amount.replace(/[^0-9]/g,'');
+    // check '' amount
+    amount!= '' ? amount : tx.amount = '0 aer';
```

Comment:
This line is a bit hard to read. Is it the same as this?
```
if (amount === '') {
 tx.amount = '0 aer';
}
```
I thought your goal was to not change the `tx` object, which I agree with.

Reply:
yes same.

just 'aer' in tx.amount occur error when send transaction to aergo node.
Throw error in function is better?